pkg/cmd/issue/edit/edit.go

```diff
@@ -107,8 +117,12 @@ func NewCmdEdit(f *cmdutil.Factory, runF func(*EditOptions) error) *cobra.Comman
 			if flags.Changed("add-project") || flags.Changed("remove-project") {
 				opts.Editable.Projects.Edited = true
 			}
-			if flags.Changed("milestone") {
+			if flags.Changed("milestone") || opts.RemoveMilestone {
```

Comment:
Should this change also be made to `gh pr edit` command?

https://github.com/cli/cli/blob/42da8baf7046b801e4c9c78a23effc048a2c81c5/pkg/cmd/pr/edit/edit.go#L119-L121

Reply:
Yeah. I'll update the `gh pr edit` command.